packages/ui-extensions/docs/surfaces/checkout/staticPages/extension-overview.doc.ts

```diff
@@ -18,8 +18,8 @@ You register for targets in your [configuration file](/docs/api/checkout-ui-exte
   sections: [
     {
       type: 'GenericAccordion',
-      title: 'Checkout placements',
-      anchorLink: 'placements',
+      title: 'Checkout locations',
+      anchorLink: 'supported-locations',
```

Comment:
Looks pretty inoffensive, and seems linked to other precedent file changes, but confirming in case @jamesvidler ?

Same question for other related changes below in this file (eg. `Thank you locations`) ?